dashboard/modules/job/sdk.py

```diff
@@ -60,6 +60,8 @@ class JobSubmissionClient(SubmissionClient):
             via a simple dict update.
         headers: Headers to use when sending requests to the HTTP job server, used
             for cases like authentication to a remote cluster.
+        verify: Boolean indication to verify the server's TLS certificate or a path to
+            a file or directory of trusted certificates.
```

Comment:
We should state the default in this docstring.

Reply:
+1

Reply:
Will clients need to explicitly set `verify` to `False` to opt-out of TLS?

Reply:
@shrekris-anyscale that's correct.

`verify=True` is the default for the `requests.request` function so default client behaviour is unchanged.